python/xorbits/web/tests/test_request_web.py

```diff
@@ -22,7 +22,7 @@
 
 @pytest.fixture
 def init_with_web():
-    sess = init_test(n_cpu=2, web=True)
```

Comment:
Why remove `n_cpu=2`?

Reply:
This case is to check if web works, seems doesn't need multi-processing cluster.